purchasing_test/integration/opportunities/test_opportunities_front.py

```diff
@@ -250,3 +252,16 @@ def test_manage_subscriptions(self):
 
         self.assert200(unsubscribe_all)
         self.assertTrue('You are not subscribed to anything!' in unsubscribe_all.data)
+
+class TestExpiredOpportunities(TestOpportunitiesBase):
+    render_templates = True
+
+    def setUp(self):
+        super(TestOpportunitiesBase, self).setUp()
+
+    def test_expired_opportunity(self):
+        '''Tests that expired view works
+        '''
+
+        expired = self.client.get('/beacon/opportunities/expired')
+        self.assert200(expired)
```

Comment:
The problem is I keep getting this context error: 
```AssertionError: Popped wrong request context.  (<RequestContext 'http://localhost/beacon/opportunities/expired' [GET] of purchasing.app> instead of <RequestContext 'http://localhost/' [GET] of purchasing.app>)```